generator/src/main/java/com/scottlogic/deg/generator/GenerateTestCases.java

```diff
@@ -116,6 +124,15 @@ public void run() {
         }
     }
 
+    private DataSetWriter getWriter(){
+        DataSetWriter outputWriter = new CsvDataSetWriter();
```

Comment:
Can we extract the writer selection to a shared place between Generate and GenerateTestCases so that if the writers we have change we don't have to change it in both places. Thanks.

Reply:
The writer selection process will eventually be calculated in the IOC/DI process, this is following the current pattern for now.